Altis_Life.Altis/config/Config_Master.hpp

```diff
@@ -95,6 +96,8 @@ class Life_Settings {
     jail_seize_vItems[] = { "spikeStrip","lockpick","goldbar","blastingcharge","boltcutter","defusekit","heroin_unprocessed","heroin_processed","cannabis","marijuana","cocaine_unprocessed","cocaine_processed","turtle_raw" }; //Define VIRTUAL items you want to be removed from players upon jailing here. Use "jail_seize_inventory" for Arma inventory items.
     jail_seize_inventory = false; //Set to true to run the cop seize script on inmates. False will remove only weapons and magazines otherwise. (Basically used in case cops forget to seize items). [See Lines 106-111 below]
     sendtoJail_locations[] = { "police_hq_1", "police_hq_2", "cop_spawn_3", "cop_spawn_5", "Correctional_Facility" }; //Enter the variableName from the mission.sqm here to allow cops to send a person to jail at these locations.
+    jail_forceWalk = true; // Disable running/jumping in jail to avoid glitchers
+    jail_helicopter_only = true; // If enabled (true) this will teleport players back into the jail if they're not in a helicopter to avoid jailglitches
```

Comment:
Do you want me only to remove the toggle or to remove the helicopter teleport back?